src/site/content/en/blog/embed-best-practices/index.md

```diff
@@ -0,0 +1,314 @@
+   allowfullscreen>
+</iframe>
+```
+Similarly, lazysizes may be used with iframes for other third-party embeds.
+
+Note that _lazysizes_ uses the [Intersection Observer API](https://developer.mozilla.org/docs/Web/API/Intersection_Observer_API) to detect when an element becomes visible. 
+
+
+#### Using data-lazy in Facebook
+
+Facebook provides different types of [social plugins](https://developers.facebook.com/docs/plugins) that can be embedded. This includes posts, comments, videos, and the most popular _Like_ button. All plugins include a setting for `data-lazy`. Setting it to `true` ensures that the plugin will use the browser's lazy-loading mechanism by setting the `loading="lazy"` iframe attribute.
+
+
+#### Lazy-loading Instagram feeds
+
+Instagram provides a block of markup and a script as part of the embed. The script injects an `<iframe>` into the page. Lazy-loading this `<iframe>` can improve performance as the embed can be over 100 KB gzipped in size. Many Instagram plugins for WordPress sites like [WPZoom](https://wordpress.org/plugins/instagram-widget-by-wpzoom/) and [Elfsight](https://www.mapledesign.co.uk/tech-blog/elfsight-instagram-feed-performance/) provide the lazy-loading option.
+
+
+### **Replace embeds with facades**
+
+While interactive embeds add value to the page, many users may not interact with them. For example, not every user browsing a restaurant page will click, expand, scroll, and navigate the map embed. Similarly, not every user to a telecom service providers page will interact with the chatbot. In these cases, you can avoid loading or lazy-loading the embed altogether by displaying a facade in its place. 
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>A map embed with a zoom in/out feature</th>
+        <th>A map facade that is an image</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/Cn0x7aeqCw7M0X5b4L1P.png", alt="A map embed", width="800", height="725" %}</td>
+        <td>{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/f8z9MfvgIFiBkCLA1Qud.png", alt="A map facade", width="800", height="541" %}</td>
+      </tr>
+    </tbody>
+    <caption>Embed vs Facade.</caption>
+  </table>
+</div>
+
+A [facade](https://web.dev/third-party-facades/) is a static element that looks similar to the actual embedded third-party but is not functional and, therefore, much less taxing on the page load. Following are a few strategies to load such embeds optimally while still providing some value to the user.
+
+#### Use static images as facades
+
+Static images can be used instead of map embeds where you might not need to make the map interactive. You can zoom in on the area of interest on the map, capture an image, and use this instead of the interactive map embed. You can also use _DevTools **Capture node screenshot** _functionality_ _to capture a screenshot of the embedded `iframe` element, as shown below.
```

Comment:
```suggestion
Static images can be used instead of map embeds where you might not need to make the map interactive. You can zoom in on the area of interest on the map, capture an image, and use this instead of the interactive map embed. You can also use DevTools **Capture node screenshot** functionality to capture a screenshot of the embedded `iframe` element, as shown below.
```